### DISCUSSION TITLE
Howto multiple foreign keys - 1-1 & 1-many, pointing to the same opposite relation?

### DISCUSSION THREAD

**Discussion by emrysal** [prisma/prisma#12919] [Q&A]
I would like a relationship like below; where the deletion of the Schedule referenced in the User under default schedule is cleared automatically. Is this possible somehow without adding the opposite relation? (The opposite relation is already defined in this case under `userId`).

```
User {
  schedules Schedule[]
  defaultSchedule Schedule
}

Schedule {
  user User
  userId Int
}
```

**Comment by pantharshit00** [comment]
Hello @emrysal , our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

If this is still relevant, can you please let me know if you are whether if you want schedule to be removed automatically if `defaultSchedule` is unset or if you want `defaultSchedule` unset if that schedule is removed from the `schedules` array. You can also probably use `prisma db pull` here if you can already express your schema in SQL in the database.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
